mozilla_django_oidc/utils.py

```diff
@@ -97,3 +100,13 @@ def add_state_and_nonce_to_session(request, state, params):
         'nonce': nonce,
         'added_on': time.time(),
     }
+
+
+def add_state_to_cookie(response, state):
+    """
+    Adds state to cookie for logout
+    """
+    # TODO: add test cookie. If no cookie, end all sessions
+    response.set_signed_cookie('oidc_state', state)
+
```

Comment:
Need a plan if user has cookies turned off. Required? Login.gov requires the `state` value for logout.

Reply:
But also if the front end needs to save the JWT for API calls, it needs to be saved in the browser.

Multiple tabs/devices are also a use case.